### DISCUSSION TITLE
LORIS v21 has expired the lorisadmin account (likely due to inactivity?)

### DISCUSSION THREAD

**Discussion by gdevenyi** [aces/Loris#7949] [Q&A] [answered]
I have inherited a LORIS instance that has been lightly used in the past year, and which I have not logged into.

I tried to login today to find:
![image](https://user-images.githubusercontent.com/3001850/149187502-fdb82c46-251a-4e6f-a0fd-73a9ed935355.png)

Could you please provide a step-by-step process to reset the account via system admin level access on the DB/LORIS host.

**Comment by ridz1208** [comment, marked as answer]
Hi @gdevenyi,

the easiest way to get past that  is the following:
1. make sure the `Active` column in the `users` MySQL table is set to `Y` for this user
2. run the following command by replacing the `user_name` value with the actual name: `DELETE FROM user_login_history WHERE UserID='user_name'`


FYI, This will obviously wipe all traces of that user in the user_login_history, so if you actually care about that data, let me know and I will find another way to get you in

**Reply by gdevenyi** [reply]
Self documenting an actual step-by-step:
1. `mysql` console on DB with root user
2. `use lorisdb;`
3. `UPDATE users SET active = "Y" WHERE UserID = "lorisadmin";`
4. `DELETE FROM user_login_history WHERE UserID='lorisadmin';`

**Comment by xlecours** [comment]
There is also `https://github.com/aces/Loris/blob/main/tools/resetpassword.php`

**Reply by gdevenyi** [reply]
Thanks. The password is known. Its the account that's been locked out.
